akka-http-core/src/test/scala/akka/http/impl/engine/parsing/ResponseParserSpec.scala

```diff
@@ -354,6 +327,21 @@ abstract class ResponseParserSpec(mode: String, newLine: String) extends AnyFree
           |
           |""" should parseToError(MessageStartError(400: StatusCode, ErrorInfo("HTTP message must not contain more than one Content-Type header")))
       }
+
+      "multiple transfer encodings in one header" in new Test {
+        """HTTP/1.1 200 OK
+          |Transfer-Encoding: fancy, chunked
+          |
+          |""" should parseToError(MessageStartError(BadRequest, ErrorInfo("Multiple HTTP Transfer-Encoding entries not supported")))
+      }
+
+      "multiple transfer encoding headers" in new Test {
```

Comment:
Ok, for having less response tests, but we should also have one for the `Transfer-Encoding` / `Content-Length` combination.

Reply:
Added